cypress/integration/vote_for_card_spec.js

```diff
@@ -11,18 +14,33 @@ describe('Vote for a card', () => {
                     console.log("done");
                 });
             });
-        });
-
-        it('displays all played cards', () => {
             cy.get('[data-cy="played-cards"]').children().its('length').should('eq', 2);
         });
 
-        // it('prompts to vote for a card', () => {
-        //     cy.get('[data-cy="vote-card"]');
-        // });
+        it('prompts to vote for a card', () => {
+            cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
+            .then(() => {
+                cy.startGame();
+                cy.request(`http://localhost:12346/playCardWord?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
+                .then(() => {
+                    cy.playCard();
+                    console.log("done");
+                });
+            });
+            cy.get('[data-cy="vote-card"]');
```

Comment:
This should go inside the `then`, under `cy.playCard()`, so that Cypress waits for that to respond before it looks for this element